juju/juju
# Add cloud watcher and wire up to cloud spec notify watcher

The watcher to inform the environ tracker worker of cloud changes is ported across to dqlite. The main change is to add the cloud watcher itself, and then wire it through to the api server layer, replacing the mongo based watcher.

## QA steps

Can only really be tested by looking at debug-log. 
bootstrap
Set logging config "juju.worker.environ=DEBUG".
Edit a local cloud yaml to add a value for storage-endpoint
Run juju update-cloud and update the controller value
debug log should have a message `reloading cloud config`

## Links

**Jira card:** JUJU-[4621]
